lib/mixins/property-effects.js

```diff
@@ -320,8 +320,7 @@ function dispatchNotifyEvent(inst, eventName, value, path) {
   if (path) {
     detail.path = path;
   }
-  // Note, since this does not bubble, it should not need to be wrapped.
-  inst.dispatchEvent(new CustomEvent(eventName, { detail }));
+  wrap(/** @type {!HTMLElement} */(inst)).dispatchEvent(new CustomEvent(eventName, { detail }));
```

Comment:
Add comment about why this needs wrap (flushing)

Reply:
Done